compiler/record-minmax/include/MinMaxComputer.h

```diff
@@ -33,8 +38,8 @@ class MinMaxComputer
   virtual ~MinMaxComputer() = default;
 
   // Child class must implement this
-  // TODO Use proper input signature
-  virtual void update_qparam(MinMaxObserver *observer) = 0;
+  virtual void
+  update_qparam(const std::unordered_map<const luci::CircleNode *, MinMaxVectors> *minmax_map) = 0;
```

Comment:
(optional) may look better with alias , something like
```cxx
using MapNode2MinMaxVectors = std::unordered_map<const luci::CircleNode *, MinMaxVectors> *;
```

Reply:
Hmm.. I'd like to avoid `using` in the header file. https://jmmv.dev/2013/12/header-files-avoid-c-using-directives.html#the-rule